<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>time和labe的使用</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>选择日期</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <item row="1" column="3">
       <widget class="QTimeEdit" name="timeEdit_2">
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>hh.mm.ss.zzz</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>QDegug</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="dateTime">
         <datetime>
          <hour>3</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>5</month>
          <day>30</day>
         </datetime>
        </property>
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>59</minute>
          <second>59</second>
          <year>2000</year>
          <month>5</month>
          <day>30</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>5</month>
          <day>30</day>
         </datetime>
        </property>
        <property name="maximumDate">
         <date>
          <year>2000</year>
          <month>5</month>
          <day>30</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::HourSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH.mm.ss.zzz</string>
        </property>
        <property name="time">
         <time>
          <hour>3</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd hh:mm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>字符串提示</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>fonrstring</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>读取当前日期</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>fonrstring</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>日期时间</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>QDegug</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>fornstring</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>QDegug</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDateEdit" name="dateEdit_2">
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDateTimeEdit" name="dateTimeEdit_2">
        <property name="displayFormat">
         <string>yyyy-MM-dd hh:mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>选择在日期</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="selectedDate">
         <date>
          <year>2019</year>
          <month>2</month>
          <day>8</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
